src/test/java/hw1/CalculatorBaseTest.java

```diff
@@ -0,0 +1,22 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class CalculatorBaseTest {
+
+    protected Calculator calculator;
+    protected static final double DOUBLE_DELTA = 1e-6;
+
+    @BeforeClass
+    public void setUp() {
+        System.out.println("213");
```

Comment:
console log?

Reply:
Yep. Forget to delete this one. Removed now